forge-gui/res/cardsfolder/upcoming/guiding_bolt.txt

```diff
@@ -0,0 +1,6 @@
+Name:Guiding Bolt
+ManaCost:2 W
+Types:Instant
+A:SP$ Destroy | Cost$ 2 W | ValidTgts$ Creature.powerGE4 | SubAbility$ DBScry | TgtPrompt$ Select target creature with power 4 or greater | SpellDescription$ Destroy target creature with power 4 or greater.
+SVar:DBScry:DB$ Scry | Cost$ 2 W | ScryNum$ 2 | SpellDescription$ Scry 2. (Look at the top two cards of your library, then put any number of them on the bottom of your library and the rest on top in any order.)
```

Comment:
No Cost here... really don't need it in line 4 either, but here it is definitely wrong

Reply:
already updated this